task_templates/3_pipelines/6_python3_sklearn_multiclass/create_pipeline.py

```diff
@@ -92,10 +92,16 @@ def to_string(x):
 # For text, we:
 # 1. Impute missing values with the string "missing"
 # 2. Tfidf encode the text, using 1-grams and 2-grams.
+
+# Sklearn's TfidfVectorizer can only handle one column of text at a time,
```

Comment:
why does this still have a text_pipeline?  is there a reason that it's not the same as the other ones?

Reply:
assuming it works, rm duplicate comment